jetty-websocket/websocket-jetty-tests/pom.xml

```diff
@@ -12,6 +12,7 @@
 
   <properties>
     <bundle-symbolic-name>${project.groupId}.tests</bundle-symbolic-name>
+    <maven.javadoc.skip>true</maven.javadoc.skip>
```

Comment:
Should not the deploy be skipped too?

Reply:
correct I forgot that

Reply:
fixed https://github.com/eclipse/jetty.project/pull/7580/commits/10a9706d6ddd9c24767759742fbf2249d8d2b028